docs/source/using/profiles.adoc

```diff
@@ -29,7 +29,7 @@ These commands are described in the following sections.
 
 There are two ways to create a new profile.
 
-=== Using The `--profile` Flag
+=== Using the `--profile` Flag
```

Comment:
I thought we should use capital letter for starting any word in a topic name. What am I missing?

Reply:
Title case is more nuanced. Essentially, capitalize all words that are not articles (a, an, the), prepositions (in, on, with), or conjunctions (and, but, or) unless they begin the title.

There are numerous versions of title case, many of them with far more rules than this -- I think that this is appropriately simple, and hopefully not too difficult to understand.